apps/web/src/server/views/app/layouts/README.md

```diff
@@ -0,0 +1,16 @@
+# NJ Layouts
```

Comment:
The reason I used _ in front of the folders for layouts is to make the distinction between what can be used in express directly vs what can be extended or included only.

I think it provides better visual separation, but I have no strong feeling if you don't want to keep the _.

Reply:
I think the _ makes sense if they coexist in the same folder as the pages, but in being in their own layouts folder, the _ is redundant in my opinion. The visual separation is now the fact they're in /layouts